cmd/workflow-controller/main.go

```diff
@@ -103,7 +103,7 @@ func NewRootCommand() *cobra.Command {
 	command.Flags().StringVar(&logLevel, "loglevel", "info", "Set the logging level. One of: debug|info|warn|error")
 	command.Flags().IntVar(&glogLevel, "gloglevel", 0, "Set the glog logging level")
 	command.Flags().IntVar(&workflowWorkers, "workflow-workers", 8, "Number of workflow workers")
-	command.Flags().IntVar(&podWorkers, "pod-workers", 8, "Number of pod workers")
+	command.Flags().IntVar(&podWorkers, "pod-workers", 32, "Number of pod workers")
```

Comment:
Why do we need to increase the default value if it is already parameterized? I am not sure we need to increase the resource for controller pod.